workflows/sarek.nf

```diff
@@ -1215,7 +1215,7 @@ workflow SAREK {
         // ANNOTATE
         if (params.step == 'annotate') vcf_to_annotate = input_sample
 
-        if (params.tools.split(',').contains('merge') || params.tools.split(',').contains('snpeff') || params.tools.split(',').contains('vep')) {
+        if (checkTools(params.tools, 'merge') || checkTools(params.tools, 'snpeff') || checkTools(params.tools, 'vep')) {
```

Comment:
Can we have `checkTools(params.tools, ['merge', 'snpeff', 'vep'])` or something?